yarn-project/stdlib/src/interfaces/aztec-node.ts

```diff
@@ -517,13 +534,16 @@ export const AztecNodeApiSchema: ApiSchemaFor<AztecNode> = {
 
   getTxEffect: z.function().args(TxHash.schema).returns(indexedTxSchema().optional()),
 
-  getPendingTxs: z.function().returns(z.array(Tx.schema)),
+  getPendingTxs: z
+    .function()
+    .args(optional(z.number().gte(1).lte(MAX_RPC_LEN).default(MAX_RPC_LEN)), optional(TxHash.schema))
```

Comment:
Shall we set the default for this to be something lower? 100 txs is still quite a lot.

Reply:
Halved it to 50